platform/features/table/res/templates/mct-table.html

```diff
@@ -43,14 +32,32 @@
                         width: columnWidths[$index] + 'px',
                         'max-width': columnWidths[$index] + 'px',
                     }">
-                    <div class="holder l-filter flex-elem grows">
-                        <input type="search"
+                    <input type="text"
                             ng-model="filters[header]"/>
-                        <a class="clear-icon clear-input icon-x-in-circle" ng-class="{show: filters[header]}" ng-click="filters[header] = undefined"></a>
```

Comment:
This was added to fix #1989 and should not be removed.  Need to restore this functionality.

Reply:
Has been resolved.